tests/specs/alone/dialInAudio.spec.ts

```diff
@@ -0,0 +1,83 @@
+import https from 'node:https';
+import process from 'node:process';
+
+import { ensureOneParticipant } from '../../helpers/participants';
+import { cleanup, waitForAudioFromDialInParticipant } from '../helpers/DialIn';
+
+describe('Dial-In - ', () => {
+    it('join participant', async () => {
+        // check rest url is configured
+        if (!process.env.DIAL_IN_REST_URL) {
+            ctx.skipSuiteTests = true;
+
+            return;
+        }
+
+        await ensureOneParticipant(ctx);
+
+        // check dial-in is enabled
+        if (!await ctx.p1.driver.execute(() => Boolean(
+            config.dialInConfCodeUrl && config.dialInNumbersUrl && config.hosts && config.hosts.muc))) {
+            ctx.skipSuiteTests = true;
+        }
+    });
+
+    it('retrieve pin', async () => {
+        const dialInPin = await ctx.p1.getInviteDialog().getPinNumber();
+
+        await ctx.p1.getInviteDialog().clickCloseButton();
+
+        if (dialInPin.length === 0) {
+            console.error('dial-in.test.no-pin');
+        }
+
+        expect(dialInPin.length >= 9).toBe(true);
+
+        ctx.dialInPin = dialInPin;
+    });
+
+    it('invite dial-in participant', async () => {
+        if (!await ctx.p1.isInMuc()) {
+            // local participant did not join abort
+            return;
+        }
+
+        const restUrl = process.env.DIAL_IN_REST_URL?.replace('{0}', ctx.dialInPin);
+
+        // we have already checked in the first test that DIAL_IN_REST_URL exist so restUrl cannot be ''
+        const responseData: string = await new Promise((resolve, reject) => {
+            https.get(restUrl || '', res => {
+                let data = '';
+
+                res.on('data', chunk => {
+                    data += chunk;
+                });
+
+                res.on('end', () => {
+                    ctx.times.restAPIExecutionTS = Date.now();
+
+                    resolve(data);
+                });
+            }).on('error', err => {
+                console.error('dial-in.test.restAPI.request.fail');
```

Comment:
Is this log lione magical somehow? Otherwise you can just let the promise fail and a traceback will be logged.

Reply:
Yep, it is. There are several logs like this that are processed from jenkins
https://github.com/jitsi/jitsi-meet-torture/blob/8452ea6416454fd3389163484889841352ab6f0b/src/test/java/org/jitsi/meet/test/DialInAudioTest.java#L236